src/controllers/users.controller.ts

```diff
@@ -60,6 +60,18 @@ class UsersController {
       next(error);
     }
   };
+
+  public manageGroups = async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const data = req.body;
```

Comment:
- Ensure that you validate request body 

e.g.
`  
    if (!req.body || !req.body.groups) {
      throw new CustomError(400, 'Invalid request body');
    }`